webpack.production.config.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
```

Comment:
you have too much copypaste, better to create 1 webpack config with dependence to env variable like `NODE_ENV`. For example:
```js
{
  // ...
  devtool: process.env.NODE_ENV === 'production' ? 'eval' : null,
  // ...
}
```